.github/workflows/Tests.yaml

```diff
@@ -227,6 +227,43 @@ jobs:
         compiler: [gcc-6, gcc-7, gcc-8, gcc-9, clang-4.0, clang-5.0,
           clang-8, clang-9]
         build_type: [Debug, Release]
+        use_pch: [OFF, ON]
```

Comment:
I think this option needs to be added to the cache key on line 316, making it `key: ccache-${{ matrix.compiler }}-${{ matrix.build_type }}-pch{{ matrix.use_pch }}`

Reply:
Okay, should be done. Hopefully GitHub handles what I did correctly... It at least accepts the syntax...

Reply:
Yes it works, in [this check](https://github.com/sxs-collaboration/spectre/pull/2134/checks?check_run_id=527749632) a cache with key `ccache-clang-9-Debug-pch-OFF` is being requested.